sdk/core/Azure.Core/tests/RecordSessionTests.cs

```diff
@@ -200,6 +201,29 @@ public void SavingRecordingSanitizesValues()
             StringAssert.DoesNotContain("secret", text);
         }
 
+        [Theory]
+        [TestCase("Content-Type")]
+        [TestCase("Accept")]
+        [TestCase("Random-Header")]
+        public void SpecialHeadersNormalizedForMatching(string name)
+        {
+            // Use HttpClientTransport as it does header normalization
+            var originalRequest = new HttpClientTransport().CreateRequest();
+            originalRequest.Method = RequestMethod.Get;
+            originalRequest.Uri.Reset(new Uri("http://localhost"));
+            originalRequest.Headers.Add(name, "application/json;odata=nometadata");
+
+            var playbackRequest = new MockTransport().CreateRequest();
+            playbackRequest.Method = RequestMethod.Get;
+            playbackRequest.Uri.Reset(new Uri("http://localhost"));
+            playbackRequest.Headers.Add(name, "application/json;odata=nometadata");
```

Comment:
Should we add at least one ignored header also to validate that logic?

Reply:
I think I'm missing something. Why do we need an ignored header here?

Reply:
Just to ensure that they are, in fact, ignored in the re-normalize logic.